h2o-hadoop/h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -1352,6 +1359,15 @@ private int run2(String[] args) throws Exception {
               "};"
       );
       addMapperConf(conf, "-login_conf", "login.conf", krbConfData);
+    } else if (pamLogin) {
+      // Use default PAM configuration file
+      final byte[] pamConfData = StringUtils.bytesOf(
+              "pamloginmodule {\n" +
+                      "     de.codedo.jaas.PamLoginModule required\n" +
+                      "     service = h2o;\n" +
+                      "};"
+      );
+      addMapperConf(conf, "-login_conf", "login.conf", pamConfData);
     }
```

Comment:
I think it makes more sense to do this [provide default login.conf for PAM] at the h2o layer rather than the h2odriver layer.
(Since you could start h2o lots of different ways.)